### PR TITLE
pip-audit 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by rouge8**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pip-audit.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
+class PipAudit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Audits Python environments and dependency trees for known vulnerabilities"
+  homepage "https://pypi.org/project/pip-audit/"
+  url "https://github.com/trailofbits/pip-audit/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "15ff2445ed2edb22dc7665366ad66563f5deee8dba0cc92e57b7ba9a421c97c6"
```
Comment (carlocab) [inline Formula/pip-audit.rb:7]:
```suggestion
  url "https://files.pythonhosted.org/packages/64/39/07bba0cff03c8126ab1ac319f00746ccc24c784ff8b4ae14963d440d4173/pip-audit-1.0.1.tar.gz"
  sha256 "351f7f279ffb20cfad3fe876e84bf524a87aaa2a8830a4cad66f24f30b2a68f6"
```
This works better with our tooling for updating resources automatically.

File: Formula/pip-audit.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+  resource "resolvelib" do
+    url "https://files.pythonhosted.org/packages/ac/20/9541749d77aebf66dd92e2b803f38a50e3a5c76e7876f45eb2b37e758d82/resolvelib-0.8.1.tar.gz"
+    sha256 "c6ea56732e9fb6fca1b2acc2ccc68a0b6b8c566d8f3e78e0443310ede61dbd37"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/pip-audit.rb:95]:
```suggestion
```
Let's replace this with `depends_on "six"`.

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @rouge8. A few suggestions.

**Comment by rouge8** [comment]
@carlocab updated with your feedback! ✅

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**